src/constants.ts

```diff
@@ -47,6 +47,10 @@ export const PUBLIC_NETWORKS: { [chainId: number]: { name: string; etherscan: st
   43114: { name: "avalanche", etherscan: "https://snowtrace.io" },
   84531: { name: "base-goerli", etherscan: "https://basescan.org" },
   421613: { name: "arbitrum-goerli", etherscan: "https://goerli.arbiscan.io" },
+  11155111: { name: "sepolia", etherscan: "https://sepolia.etherscan.io" },
+  84532: { name: "base-sepolia", etherscan: "https://sepolia.basescan.org" },
+  534352: { name: "scroll", etherscan: "https://scrollscan.com" },
+  534351: { name: "scroll-sepolia", etherscan: "https://sepolia.scrollscan.com" },
```

Comment:
It looks like the keys in this mapping are sorted numerically - OK to maintain that with this change?

Reply:
Good call. I've made this numerically sorted and added more outstanding block explorer links.